e2e/journey-suites/src/protect.test.ts

```diff
@@ -9,32 +9,42 @@ import { expect, test } from '@playwright/test';
 import { asyncEvents } from './utils/async-events.js';
 import { username, password } from './utils/demo-user.js';
 
-test.skip('Test happy paths on test page', async ({ page }) => {
-  const { clickButton, navigate } = asyncEvents(page);
-  await navigate('/?journey=TEST_Protect');
+test('Test PingOne Protect journey flow', async ({ page }) => {
+  const { clickButton } = asyncEvents(page);
 
   const messageArray: string[] = [];
 
-  // Listen for events on page
   page.on('console', async (msg) => {
     messageArray.push(msg.text());
     return Promise.resolve(true);
   });
 
-  // Perform basic login
+  await page.goto('/?journey=TEST_LoginPingProtect&clientId=basic', { waitUntil: 'load' });
+
+  await expect(page.getByText('Initializing PingOne Protect...')).toBeVisible({ timeout: 10000 });
+  await expect(page.getByText('PingOne Protect initialized successfully!')).toBeVisible({
+    timeout: 15000,
+  });
+
+  await expect(page.getByLabel('User Name')).toBeVisible({ timeout: 15000 });
   await page.getByLabel('User Name').fill(username);
   await page.getByLabel('Password').fill(password);
   await clickButton('Submit', '/authenticate');
 
-  await expect(page.getByText('Collecting protect data')).toBeVisible();
+  await expect(page.getByText('Evaluating risk assessment...')).toBeVisible({ timeout: 10000 });
+  await expect(page.getByText('Risk assessment completed successfully!')).toBeVisible({
+    timeout: 15000,
+  });
```

Comment:
Same as the Device Profile test. Let's test the request body directly, rather than just testing for messages.